crates/rpc/rpc-types/src/eth/fee.rs

```diff
@@ -20,3 +21,21 @@ pub struct FeeHistory {
     #[serde(default)]
     pub reward: Option<Vec<Vec<U256>>>,
 }
+
+/// LRU cache for `eth_feeHistory` RPC method. Block Number => Fee History.
+pub type FeeHistoryCache = LruCache<BlockNumber, FeeHistoryCacheItem>;
```

Comment:
I really like to wrap these things in a new type instead of using aliases.

Also we probably will populate this from a different task so this type should wrap `Arc<Mutex<..>>` directly

Reply:
fixed

Reply:
currently, we're not populating this cache from a different task, but the concurrent access is still possible due to cache population in the RPC call, so we still need the `Arc<Mutex<..>>`